counter-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: ticktock
```

Comment:
This is something we need to determine in general - what is the convention or do we even need to supply a name here vs generating a default (e.g. application.name)? We do need to keep in mind that a spring-cloud-stream module should eventually be runnable as a self-contained app with non-spring-cloud-stream app(s) interacting with it from the other side of the broker.

We also need to change from "ticktock" in the log-sink module since that should likewise be a general purpose app. That was in place since we only had 2 modules "time" and "log".